profile/README.md

```diff
@@ -0,0 +1,12 @@
+Valkey is an open source, in memory datastore released under the [BSD-3 Clause License](https://github.com/valkey-io/valkey?tab=BSD-3-Clause-1-ov-file#readme). It is a continuation of the work that was being done on Redis 7.2.4. 
+
+## Community 🎉
+
+- 🎙️ Chat with us on [Discord](https://discord.gg/zbcPa5umUB)
+- ❔ Also there is an active community on [Stack Overflow](https://stackoverflow.com/questions/tagged/redis)
```

Comment:
I'm not sure we should leave the SO redis tag.

Reply:
😛  I felt like it might be nice in the interim as there are likely no Valkey tagged SO posts.

Reply:
I get what you're saying but , it feels a bit like claiming the `redis` community on SO is `valkey`'s